### ISSUE TITLE
Possible to make client service of type LoadBalancer?

### ISSUE THREAD

**Issue by Raboo** [pravega/zookeeper-operator#416]
Hi

Is there a way to make the clientService become of type LoadBalancer.

I tried setting helm values to
```
clientService:
  spec:
    type: LoadBalancer
```

But that didn't bite. Service is still ClusterIP. I want to access the zookeeper cluster from outside of k8s internal network.

**Comment by Raboo** [comment]
@anishakj Ok, can  you add the possibility to set the client service as type LoadBalancer?

**Comment by Raboo** [comment]
I just realized that this is more difficult than adding a service of type LoadBalancer as the client uses it for discovery and then tries to speak to the zookeeper nodes directly, which in this case they are in k8s internal network.

```
New config event received: {server.1=zookeeper-0.zookeeper-headless.test.svc.cluster.local:2888:3888:participant;0.0.0.0:2181, version=100000010, server.3=zookeeper-2.zookeeper-headless.test.svc.cluster.local:2888:3888:participant;0.0.0.0:2181, server.2=zookeeper-1.zookeeper-headless.test.svc.cluster.local:2888:3888:participant;0.0.0.0:2181}
```

So I guess each StatefulSet needs a service with type LoadBalancer and then zookeeper must be configured to use those services(ip address, consul/svc discovery or externaldns)

**Comment by anishakj** [comment]
@Raboo that is correct understanding. We don't have  bandwidth to implement this feature currently. Would you like to create a PR to add this support

**Comment by Raboo** [comment]
@anishakj If I could I would, I'm not really a programmer, don't know much go-lang.

But I assume that the implementation logic would look something like this:
- _Service_ zookeeper-client as _LoadBalancer_
- each pod get it's own _Service_ for example POD-0, POD-1, POD-2 as _LoadBalancer_. _Service_ status _ExternalIP_ gets populated with a routed IP address ex. 10.0.0.1, 10.0.0.2, etc..
- operator fetches these IP's and populates zookeeper cfg.
- zookeeper cfg gets server.1=10.0.0.1, server.2=10.0.0.2... etc.
